app/models/manageiq/providers/autosde/storage_manager/cloud_volume.rb

```diff
@@ -17,22 +17,15 @@ def self.raw_create_volume(ext_management_system, options = {})
       :count   => options["count"]
     )
 
-    new_volume = ext_management_system.autosde_client.VolumeApi.volumes_post(vol_to_create)
-
-    EmsRefresh.queue_refresh(
-      InventoryRefresh::Target.new(
-        :manager     => ext_management_system,
-        :association => :cloud_volumes,
-        :manager_ref => {:ems_ref => new_volume.uuid}
-      )
-    )
+    task_id = ext_management_system.autosde_client.VolumeApi.volumes_post(vol_to_create).task_id
+    ext_management_system.class::EmsRefreshWorkflow.start_refresh(:cloud_volumes, "new", ext_management_system.id, task_id)
```

Comment:
I think we'd be better off passing `nil` rather than `"new"` here because there is code which will do a database query using `target_class+target_id` which would end up doing `CloudVolume.find("new")`